datafusion/sql/src/expr/mod.rs

```
@@ -271,8 +271,14 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
             ),
 
             SQLExpr::Cast {
-                expr, data_type, ..
+                expr,
+                data_type,
+                format,
```

previously `format` would be silently ignored